pkg/operator/target_config_reconciler.go

```diff
@@ -880,6 +880,36 @@ func (c *TargetConfigReconciler) manageDeployment(kueueoperator *kueuev1alpha1.K
 		required.Spec.Template.Spec.Containers[0].VolumeMounts,
 		metricsCertVolumeMount,
 	)
+
+	configVolume := v1.Volume{
+		Name: "kueue-config",
+		VolumeSource: v1.VolumeSource{
+			ConfigMap: &v1.ConfigMapVolumeSource{
+				LocalObjectReference: v1.LocalObjectReference{
+					Name: configMap.Name,
+				},
+			},
+		},
+	}
+	required.Spec.Template.Spec.Volumes = append(required.Spec.Template.Spec.Volumes, configVolume)
```

Comment:
I think that the existing deployment already has this no?

Should there be a replace rather than an append?